drivers/wifi/infineon/airoc_wifi.h

```diff
@@ -34,11 +62,26 @@ struct airoc_wifi_data {
 	uint8_t frame_buf[NET_ETH_MAX_FRAME_SIZE];
 };
 
+union airoc_wifi_bus {
+#if defined(CONFIG_AIROC_WIFI_BUS_SDIO)
+	const struct device *bus_sdio;
+#endif
+#if defined(CONFIG_AIROC_WIFI_BUS_SPI)
```

Comment:
Same suggestion as above. Is this an either/or thing, or have I misunderstood the implementation?

Reply:
Likewise.